README.MD

```diff
@@ -1,6 +1,14 @@
 # Odysseus
 A Project Odyssey tool for converting FTB and HQM quest-packs to the Heracles format.
 
+## Installation
+### Prerequisites: Node.js, npm
+
+1. In the main repo directory, do `npm install`.
+2. In the odysseus sub-directory, do `npm run build`.
```

Comment:
`npm run build` should be done from the root directory with `--workspace=dir` options or a `--workspaces` option, for example `npm run build --workspace=odysseus --workspace=odysseus-cli`.

Reply:
Thank you; I think Clawby/Red already wrote a more complete version. I'll close this PR soon then.